mlflow/sklearn/__init__.py

```diff
@@ -188,12 +188,14 @@ def save_model(
     with open(os.path.join(path, conda_env_subpath), "w") as f:
         yaml.safe_dump(conda_env, stream=f, default_flow_style=False)
 
-    pyfunc.add_to_model(
-        mlflow_model,
-        loader_module="mlflow.sklearn",
-        model_path=model_data_subpath,
-        env=conda_env_subpath,
-    )
+    # `PyFuncModel` only works for sklearn models that define `predict()`.
```

Comment:
Made a minor tweak to this comment to remove the double negative

Reply:
makes sense